docs/slurk_layouts.rst

```diff
@@ -153,6 +153,7 @@ Examples:
   - ``"enforce-fullscreen"``: Page content is grayed out, until a button is clicked that sends user into fullscreen. See ``enforce-fullscreen_layout.json`` in ``examples`` for an example layout
   - ``"bounding-boxes"``: Makes it possible for users to draw rectangles inside a designated drawing area (html element with the id ``drawing-area``). Per default, drawn rectangles are not shared between users inside a room. If you wish for all users in a room to share a common canvas give all users inside the room the permission ``receive_bounding_box``
   - ``"mouse-tracking"``: Mouse movement and clicks inside the designated html element with the id ``tracking-area`` are registered. They can be handled by bots through the ``mouse`` event.
+  - ``"live-typing"``: Chat partners receive a preview of messages that are currently being typed. Typed messages can't be edited in this mode. If a user started a message and stopped typing for 3sec, it is automatically submitted.
```

Comment:
isn't that the 3 that you changed to a 5 in https://github.com/clp-research/slurk/pull/234/commits/6972091ef8e510023816607c5382a5496547209c?

Reply:
This PR includes the two commits from the typing-users PR and a third commit for live-typing that reverts this change. Information about the typing is now included in the chat history. If a new message is added to the chat history, the chat automatically scrolls down to it. I didnt wish to enforce the same behaviour for typing, because I think this would make it nearly impossible to read through the chat history. If a user was currently scrolling through the history they could have easily missed the typing bubble if it only shows up for a short time. That's why I considered increasing the time it pops up. But for live-typing it is better for messages to be submitted as timely as possible, as the preview is not logged. So I reverted it.

Reply:
but what does that mean for the other PR? Can I merge it?

Reply:
I reverted the change also for the other PR, so that it can be merged independently of this one here. So yes.